pkg/services/ngalert/sender/router.go

```diff
@@ -27,11 +27,11 @@ type AlertsRouter struct {
 	clock            clock.Clock
 	adminConfigStore store.AdminConfigurationStore
 
-	// senders help us send alerts to external Alertmanagers.
-	adminConfigMtx sync.RWMutex
-	sendAlertsTo   map[int64]models.AlertmanagersChoice
-	senders        map[int64]*Sender
-	sendersCfgHash map[int64]string
+	// externalAlertmanagerSenders help us send alerts to external Alertmanagers.
+	adminConfigMtx                     sync.RWMutex
+	sendAlertsTo                       map[int64]models.AlertmanagersChoice
+	externalAlertmanagerSenders        map[int64]*ExternalAlertmanager
+	externalAlertmanagerSendersCfgHash map[int64]string
```

Comment:
```suggestion
	externalAlertmanagers        map[int64]*ExternalAlertmanager
	externalAlertmanagersCfgHash map[int64]string
```

Reply:
I did not rename them because in struct sender looked more descriptive. I am fine with these names, though